config/rollup.base.js

```diff
@@ -62,7 +65,7 @@ export const getBanner = ({ repository }, buildMetadata) => {
   );
 
   const text = `
-  FPC (Forklift Plugin Components)
+  KFC (Kubernetes Forklift Components)
```

Comment:
This name isn't used anywhere else right?  I suggest keeping it the same name as everything else in the repo, at least as a base.  Maybe add `name` from the package.json file and do it as:

```js
const test = `
  @kubev2v/forklift-console-plugin (package: ${name})
  ...
```

Reply:
> This name isn't used anywhere else right?

The `getBanner` method is used to create a comment on the top of each file created by rollup, the banner will appear at the top of the file. Banner comments at the top of the file are usually used for license information, Usage Instructions, brief description of what the file does and metadata about the file.

In our case we will use module bundlers, webpack and rollup to compile and pack our code into efficient bundles for consumption in other projects. Bundeling the code may include minification and optimization of JavaScript files. Minification is the process of removing all unnecessary characters from the source code without changing its functionality.

Adding a banner on the top of the minified file may help to preserving metadata and licensing information. This top comment will appear on the top of bundled files.

> I suggest keeping it the same name as everything else in the repo, at least as a base. Maybe add name from the package.json

The comment banner suggested here will include metadata about the file, such as the author, date of creation, last update, version, and other relevant information. This can be useful for version control and understanding the history and context of the file. `KFC` serve this purpose. Using the projects name field may also serve similar purpose.

`KFC`
pros:
a. Uniqueness: this string is not used elsewhere in the project, it will help to search for in debugging situation, and be easily destinguashable from comments set in the uncompiled source code.
b. `KFC` is also a name a fast food restaurant, using an acronym with more then one meaning like `MTV` can be fun.
cons:
a. Uniqueness: understanding the history and context of the file may be easier when using the same name all over the project.
b. KFC` is also a name a fast food restaurant, and may be hard on developers with less knowledge of food industry.

It's good to have a bit of fun with your work, but not at the expense of readability, inclusivity, and professionalism. When considering the audience of this compiled files will include our development and QE team, and the clarity needed to identify the files as part ot the Forklift project both options are posible.

I tend to agree with the suggestion of using the name from `package.json` files for it's clarity and for better keeping the projects context.